packages/storybook/src/nimble/table-column/menu-button/table-column-menu-button.mdx

```diff
@@ -32,3 +33,23 @@ responsibility to do the following:
    `newState` is `true`.
 2. Update the data through a `setData()` call when a menu item is activated in
    order to update the text on the menu button in the column.
+
+### Menu Button Column vs Table Action Menu
```

Comment:
It would be good to link to this guidance from our docs on the table action menu. Unfortunately it looks like we don't actually have docs on the table action menu? From what I can find we document the slots / attributes for it but don't have a specific example for it.

The best place I can find would be a link from the table story's "Action menus" slot.